mlir_graphblas/cli.py

```diff
@@ -49,8 +49,7 @@ def _read_input(self, file) -> bytes:
         elif hasattr(file, "close"):
             return file.read()
         else:
-            with open(file, "rb") as f:
-                return f.read()
+            raise TypeError("file must be bytes or a file object")
 
     def apply_passes(self, file, passes: List[str]) -> Union[str, "DebugResult"]:
```

Comment:
It might be helpful to add a type hint for `file`. Something like `Union[bytes, str, io.IOBase]` would work. It's redundant with the doc string, but I personally read the type hints before reading the doc string since type hints are more easily humanly-parsable and has less visual noise (IMO).

Reply:
Good idea.  I will fix.

Reply:
Should be fixed now.